evaluation/README.md

```diff
@@ -24,3 +24,18 @@ This example uses a very small, fake dataset meant to show working code. Suggest
 
 1. A groundtruth evaluation dataset relevant to the application's domain
 2. A large search dataset such as [MS Marco](https://microsoft.github.io/msmarco/)
+
+## E2E Samples
+
+This section outlines different evaluation E2E samples.
```

Comment:
I'd like @ra1han to chime in, but I'm a bit hesitant about adding more complexity to the directory structure. instead of e2e_samples, could we follow a similar structure to the `trec-evaluation.ipynb` notebook where we assume somebody created a vector db from the code samples (in this case Azure AI Search) with a dataset (in this case the chunked playbook dataset) and focus more on the set up for evaluating the data?

could be convinced to keep the more complex directory set up, but I'd like @ra1han  to give his 2 cents on it as well.

Reply:
Dropping folder structure suggested by @ra1han for history from IM:

```
I am proposing the following folder strcuture for the evaluation section. Please let me know your thoughts on this (tag Jose) -

- Rename root folder for consistency evaluation => evaluation_samples
- On the Readme on evaluation_samples, two subsections that provide introduction to two evalution sample we have today - 1
      - Evaluating vector search using TREC
      - Evaluating document chunking
- Two subfolders under evaluation_samples to represent two samples. Subfolders -
      - evaluation_trec
      - evaluation_doc_chunking
```

Reply:
This has been addressed in this commit:
https://github.com/Azure-Samples/azure-vector-database-samples/pull/44/commits/bc9fa9616b02ecf3b78703db74046911e8ab5d07